src/middleware/validationRules.ts

```diff
@@ -0,0 +1,39 @@
+import { body, param } from "express-validator";
+import { databaseModel } from "../model";
+
+export const validatePostRoute = [
+  body("originalUrl")
+    .isURL()
+    .notEmpty()
+    .withMessage("original url must be a valid URL")
+    .custom(() => {
+      return true;
+    }),
+
+  body("customName")
```

Comment:
This validation does not allow custom name with spaces to pass.

Trying to save this creates an error

```
const requestBody = {
    "originalUrl": "https://www.google.com/search?q=google+url+shortener&oq=google+url+shortener&aqs=chrom",
    "customName": "sheesh hdndbh dhs"
}.

const error = {
    "errors": [
        {
            "message": "custom name must contain only alphabetic characters",
            "path": "customName"
        }
    ]
}
```